synthesizer/photoionisation/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .photoionisation import *
```

Comment:
Why are you importing everything and then importing cloudy17 explicitly below? We should avoid * imports in favour of clarity.

Reply:
Also we shouldn't be using relative imports